test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/HapiTxnOp.java

```diff
@@ -423,13 +423,6 @@ private Response statusResponse(HapiApiSpec spec, Query receiptQuery) throws Thr
 				}
 			}
 		}
-		if(response.getTransactionGetReceipt().getReceipt().getStatus() == UNKNOWN) {
-			log.error(" [ accountID {} validStart({}, {}) ], status resolution failed with unrecognized exception, not getting txn response back",
-					receiptQuery.getTransactionGetReceipt().getTransactionID().getAccountID().getAccountNum(),
-					receiptQuery.getTransactionGetReceipt().getTransactionID().getTransactionValidStart().getSeconds(),
-					receiptQuery.getTransactionGetReceipt().getTransactionID().getTransactionValidStart().getNanos()
-					);
-		}
```

Comment:
I would like to move these lines one level above after the `do... while` loop in `resolvedStatusOfSubmission`. And this should give JRS validator better info to decide whether to fail the test or not.

Reply:
Why would we do that? After the `do... while` is a `return UNKNOWN;` then it will follow the workflow of handling response code in the caller function of `resolvedStatusOfSubmission`.

Reply:
No particular reason, just try to provide info on what might happen at the moment we know best. To my understanding, `UNKNOWN` here means that hedera server hasn't reached its final decision as whether this transaction should be marked as success or not. (Of course, UNKNOWN could also be this this transaction's final status as well.) This extra piece of information is to mainly to provide info to help JRS validator to evaluate the overall  status for perf tests.

Reply:
@ljianghedera, my point is the caller of `resolvedStatusOfSubmission`, which mostly is `resolveStatus` (for non-perf tests), has its own way to handle UNKNOWN response code already
https://github.com/hashgraph/hedera-services/blob/2e8bbe6f0b2f328ffd91cb1ae9ab644aeb91e41d/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/HapiTxnOp.java#L250-L253
If we don't set `expectedStatus` to UNKNOWN (which is 100% true as of this time of writing), we will log and throw an exception as above
https://github.com/hashgraph/hedera-services/blob/2e8bbe6f0b2f328ffd91cb1ae9ab644aeb91e41d/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/HapiTxnOp.java#L110-L112
Besides, for performance tests with the caller of `resolvedStatusOfSubmission` to be `measureConsensusLatency`, the client log will still be able to get flooded with the proposed error log.

I still think moving this error log, instead of completely removing it, is troublesome and doesn't add any value.
@tinker-michaelj, do you have any other insight or comment on my argument?

Reply:
https://hedera-hashgraph.slack.com/archives/CMD3V6ZC4/p1612294448038100
`master` GCP CI failed because of these error log entries.

Reply:
The reason master GCP CI failed because we tightened the criteria of HAPIClient validation. For any non-perf tests, these message is treated as failure. Only for perf tests, HAPIClient validation will allow small percentage of errors caused by server couldn't process quick enough. Here, there is UmbrellaRedux test in the suite with unlimited maxOpsPerSec running for one minute, that is the real cause for the test to fail.

Reply:
I think it's hard to reach a clear-cut conclusion. The change is fine with me. I guess we don't need to waste too much time on this small places. We can come back to revisit it when we have better ideas on how to handle it.